lib/gds_api/publishing_api_v2.rb

```diff
@@ -92,10 +124,28 @@ def discard_draft(content_id, options = {})
     post_json!(discard_url(content_id), params)
   end
 
+  # FIXME: Add documentation
+  #
+  # @see https://github.com/alphagov/publishing-api/blob/master/doc/publishing-api-syntactic-usage.md#get-v2linkscontent_id
   def get_links(content_id)
     get_json(links_url(content_id))
   end
 
+  # Patch the links of a content item
+  #
+  # @param content_id [UUID]
+  # @param payload [Hash] A "links hash"
```

Comment:
Ok now you're just creating work :)

I can link out to https://github.com/alphagov/publishing-api/blob/master/doc/semantics-of-the-publishing-api.md#links. @tuzz do you know about other places where `links` are documented?